scripts/ci/create-plunker-bundle.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * This script creates a bundle of all components and publishes it to Firebase.
+ * The bundle will be used to load a Plunker Demo of Angular Material 2.
+ */
+
+const spawn = require('child_process').spawnSync;
+const exec = require('child_process').execSync;
+const firebase = require('firebase-tools');
+const path = require('path');
+const fse = require('fs-extra');
+const inlineResources = require('../../tools/inline-resources-tools');
+
+const distRoot = path.resolve(process.cwd(), 'dist');
+const deployRoot = path.join(distRoot, 'plunker-deploy/');
+const mainFile = path.join(distRoot, 'main.js');
+const latestTag = getLatestTag();
+const isRelease = getSHAFromTag(latestTag) === getLatestSHA();

```

Comment:
nit: `getShaFromTag` and `getLatestSha` (they're camelCase even if they're acronyms)